manifests/g/GnuWin32/Flex/2.5.4a-1/GnuWin32.Flex.yaml

```
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: GnuWin32.Flex
+PackageVersion: 2.5.4a-1
```

```suggestion
PackageVersion: 2.5.4a
```
Please update file name and path to match this change.